### PR TITLE
Fixed an issue with the frame count management for the volumetric fog (case 1299251).

### PR DESCRIPTION
**Pull request by anisunity**
https://fogbugz.unity3d.com/f/cases/1299251/
After a recent change related to the frame count management. The Frame count was not updated properly anymore in the scene view editor.  Which lead to a various set of issues related to the volumetrics. By using the camera frame count we guarantee that the frame count is incremented at every render with that camera.

Testing status
The issue doesn't repro locally anymore.
Local tests are green, except for 9402 AlphaToMask which doesn't have to do anything with the code changed...

### CODE REVIEW

File: com.unity.render-pipelines.high-definition/Runtime/Lighting/VolumetricLighting/VolumetricLighting.cs [hunk 1]
```diff
@@ -255,6 +255,12 @@ static internal void SafeDestroy(ref RenderTexture rt)
             }
         }
 
+        static uint VolumetricFrameIndex(HDCamera hdCamera)
+        {
+            // Here we do modulo 14 because we need the enable to detect a change every frame, but the accumulation is done on 7 frames (7x2=14)
+            return hdCamera.GetCameraFrameCount() % 14;
```
Comment (sebastienlagarde) [inline com.unity.render-pipelines.high-definition/Runtime/Lighting/VolumetricLighting/VolumetricLighting.cs:261]: don't we have any constant value somewhere with this 7 ? Afraid to have hardcoded value hidden in a function like this.

Comment (anisunity) [reply com.unity.render-pipelines.high-definition/Runtime/Lighting/VolumetricLighting/VolumetricLighting.cs:261]: right now, no when the thing is accumulation frame index is computed its explicitely written frameIndex % 7

**Review by adrien-de-tocqueville — APPROVED** [review]
looks good, but the frame count management in HDRenderPipeline is still broken, so we probably still have problem with other effects using accumulation like TAA

**Review by TomasKiniulis — APPROVED** [review]
Tested repro project and template scene with Always Refresh toggled On/Off, and various Scene/Game view layouts in Playmode and edit mode. Seems to be working correctly now

**Review by sebastienlagarde — APPROVED** [review]

**Pull request merged**